i18n/po/i18n_nl.po

```diff
@@ -193,7 +238,8 @@ msgid "Schema {0} class definition {1} ancestor {2} does not exist in schema"
 msgstr ""
 
 #: ../bmm/src/main/java/org/openehr/bmm/persistence/validation/BmmMessageIds.java:67
-msgid "Schema {0} class definition {1} container property {2} cardinality not defined (assuming {0..*})"
+#, java-format
+msgid "Schema {0} class definition {1} container property {2} cardinality not defined (assuming '{0..*}')"
 msgstr ""
```

Comment:
Not translated. Also we know the default cardinality of all fields, so we can get rid of this message by changing some code.

Reply:
I have only added the translations for the RM Object Validator messages. Changing the code is out of scope for this request.